librz/asm/arch/include/opcode/mips.h

```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005, 2008 Free Software Foundation, Inc.
+// SPDX-License-Identifier: GPL-1.0-or-later
```

Comment:
Looks like this file is very old and should be updated. Latest GDB uses GPLv3-based one...
See https://github.com/rizinorg/rizin/issues/209

Reply:
I know... of course for another PR. THis one is becoming tiring already this way :(

Reply:
Yes, I wan't implying to do that, just never noticed before and worth noting.